### ISSUE TITLE
Make all data types in builtin implement Ord

### ISSUE THREAD

**Issue by chadaustin** [cruxlang/crux#135]

**Comment by dustyleary** [comment]
While it's still young, consider revisiting naming decisions.  I think
'Ordered' is better ftux than 'Ord', and the ftux-vs-experience user
tradeoff is very small.

I think the only downsides for experienced users are the fact that the name
would not be the same as Haskell's, and the slightly longer name?





-D

**Comment by chadaustin** [comment]
Good point.  Ordered is probably a better name.  For what it's worth, Rust uses the name Ord too.  But it really only matters when spelling it out in annotations which Crux avoids most of the time.
